tests/auth_basic_test.py

```diff
@@ -269,26 +300,26 @@ def test_login_invalid_credentials(client, capture_templates, setup_logger):
         client: The test client used for making requests.
         capture_templates: Context manager to capture templates rendered.
         setup_logger: Setup logger.
+        email: test email to be used in the signup form.
+        password: test password to be used along with the email in the signup
+                form.
+        http_status: Expected HTTP status code to be returned after submitting
+                the signup form with an invalid email
     """
     logger = setup_logger("test_login_invalid_credentials")
-    try:
-        response = client.post(
-            "/login",
-            data={"email": "test@example.com", "password": "wrongpassword"},
-            follow_redirects=True,
-        )
-        assert response.status_code == 200
-        assert (
-            b"Please check your login details and try again." in response.data
-        )
-        final_template_rendered = len(capture_templates) - 1
-        assert (
-            capture_templates[final_template_rendered][0].name == "login.html"
-        ), "Wrong template used"
-        logger.info("Login failed successfully with invalid credentials.")
-    except AssertionError as e:
-        logger.error(f"Test failed: {str(e)}")
-        raise
+    response = client.post(
+        "/login",
+        data={"email": email, "password": password},
```

Comment:
This test should only have an `email` and `password` parameter.

Reply:
updated!